copado-function/app/Commit.js

```diff
@@ -42,6 +42,7 @@ const CONFIG = {
     configFilePath: '.mcdevrc.json',
     debug: process.env.debug === 'true' ? true : false,
     localDev: process.env.LOCAL_DEV === 'true' ? true : false,
+    isMcdevInstalledGlobally: process.env.isMcdevInstalledGlobally === 'true' ? true : false,
```

Comment:
In the code context, this name does not make much sense, "isMcdevInstalledLocally", it never is right? At max we want to install it locally if it is not installed globally. Maybe "installLocally" or something like it would be a better name for this env variable if we want to install it locally.

We are comparing "isMcdevInstalledLocally", if yes we install it locally? This will never be true, because it is never installed locally right?